### PR TITLE
New Instrument Cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
PR includes cal csvs for instruments returning from vendor refurb, plus a glider NUTNR post-deploy (Irminger 8)...and added 2 SUMO NUTNR post-deploy cal csvs (Irminger 8)

Additionally there are 2 ADCPAM cal csvs for glider ADCP SNs that had no existing cal csvs (but are in sensor bulk load).  I used the start date of the earliest cruise these were deployed on:
CGINS-ADCPAM-44067 - first deployment on glider 339 on SI20150110
CGINS-ADCPAM-49653 - first deployment on glider 389 on same cruise - SI20150110

### CODE REVIEW

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20210824.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
+NTR-1091,CC_lower_wavelength_limit_for_spectra_fit,217,constant
+NTR-1091,CC_upper_wavelength_limit_for_spectra_fit,240,constant
```
Comment (IDCalTech) [inline calibration/NUTNRB/CGINS-NUTNRB-01091__20210824.csv:7]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01094__20210824.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+NTR-1094,CC_lower_wavelength_limit_for_spectra_fit,217,constant
+NTR-1094,CC_upper_wavelength_limit_for_spectra_fit,240,constant
```
Comment (IDCalTech) [inline calibration/NUTNRB/CGINS-NUTNRB-01094__20210824.csv:7]: Looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00661__20210814.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+661,CC_lower_wavelength_limit_for_spectra_fit,217,
```
Comment (IDCalTech) [inline calibration/NUTNRM/CGINS-NUTNRM-00661__20210814.csv:6]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20210729.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+ACS-130,CC_tbins,"[0.913569, 1.290121, 2.452333, 3.461538, 4.517059, 5.488800, 6.480417, 7.494583, 8.490800, 9.503333, 10.486667, 11.494783, 12.512800, 13.496522, 14.497500, 15.471250, 16.485000, 17.490833, 18.474815, 19.471667, 20.508400, 21.461429, 22.477619, 23.536667, 24.436897, 25.498000, 26.551000, 27.443750, 28.521905, 29.479063, 30.470500, 31.564333, 32.456250, 33.567857, 34.407667]",
+ACS-130,CC_taarray,SheetRef:CC_taarray,
+ACS-130,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (asmith-whoi) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20210729.csv:9]: Good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20210729__CC_taarray.ext [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+0.002707,0.002488,0.002651,0.002703,0.002513,0.002678,0.002562,0.002662,0.002668,0.002474,0.002559,0.002506,0.002532,0.002212,0.001948,0.001781,0.001654,0.001393,0.001189,0.000899,0.000674,0.000617,0.000505,0.000169,5e-05,0,-0.000165,-0.000253,-0.000615,-0.000553,-0.000788,-0.000799,-0.001145,-0.0014,-0.001267
+0.00261,0.002428,0.002608,0.002737,0.002613,0.002597,0.002556,0.002662,0.002677,0.002411,0.002524,0.002477,0.002479,0.002257,0.001959,0.001784,0.001685,0.001398,0.001191,0.000934,0.000652,0.000653,0.000424,0.000221,5.9e-05,0,-0.000209,-0.000341,-0.000557,-0.000551,-0.000769,-0.000808,-0.001153,-0.001441,-0.001336
+0.003078,0.002748,0.002875,0.002971,0.002813,0.002913,0.002751,0.002844,0.002841,0.002714,0.002751,0.002728,0.002665,0.002454,0.00216,0.002026,0.001749,0.001543,0.001297,0.001024,0.000801,0.00072,0.000498,0.00021,0.00013,0,-0.000205,-0.000339,-0.000514,-0.000554,-0.00088,-0.000814,-0.001162,-0.001403,-0.001256
```
Comment (asmith-whoi) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20210729__CC_taarray.ext:83]: Good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20210729__CC_tcarray.ext [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+-0.003533,-0.001992,-0.001148,-0.001183,-0.000991,-0.000998,-0.000983,-0.000958,-0.000639,-0.000686,-0.000543,-0.000493,-0.000408,-4e-05,-7.3e-05,7.6e-05,0.000172,0.000193,0.000313,0.000166,0.000304,0.000373,0.000385,0.000241,0.000133,0,-0.000127,-0.000281,-0.000329,-0.000375,-0.000663,-0.000834,-0.000902,-0.001141,-0.001244
+-0.003977,-0.002429,-0.001469,-0.001399,-0.001283,-0.001175,-0.001083,-0.001063,-0.000961,-0.000729,-0.00048,-0.00027,-0.000174,0.000127,0.000187,0.000147,0.000412,0.000453,0.000494,0.000446,0.000555,0.000254,0.000285,0.000286,0.000126,0,3.5e-05,-0.000171,-0.000441,-0.000437,-0.000777,-0.00087,-0.001006,-0.001317,-0.001462
+-0.003639,-0.002027,-0.001124,-0.000947,-0.000877,-0.000796,-0.00071,-0.000721,-0.000523,-0.00037,-0.000212,-0.000133,-0.000181,0.000149,0.000255,0.000152,0.000468,0.000397,0.000419,0.00032,0.000418,0.000272,0.000228,0.000171,8.8e-05,0,-0.000131,-0.000261,-0.000398,-0.000574,-0.000646,-0.000903,-0.000879,-0.001171,-0.001203
```
Comment (asmith-whoi) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20210729__CC_tcarray.ext:83]: Good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20210811.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+ACS-256,CC_tbins,"[0.875510, 1.244552, 2.441071, 3.448571, 4.585500, 5.505000, 6.496400, 7.493750, 8.489200, 9.502083, 10.513846, 11.505455, 12.505769, 13.487826, 14.511667, 15.478800, 16.495417, 17.475833, 18.515357, 19.496400, 20.510455, 21.512069, 22.466667, 23.552400, 24.451034, 25.519474, 26.493750, 27.491429, 28.535862, 29.453636, 30.546923, 31.444828, 32.532632, 33.496905, 34.473333]",
+ACS-256,CC_taarray,SheetRef:CC_taarray,
+ACS-256,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (asmith-whoi) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20210811.csv:9]: Good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20210811__CC_taarray.ext [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+-0.011634,-0.007327,-0.004856,-0.004438,-0.003833,-0.003332,-0.002664,-0.00212,-0.001678,-0.001076,-0.000679,-0.00021,0.000151,0.000393,0.000602,0.000847,0.000897,0.001117,0.001174,0.00106,0.001057,0.000971,0.000736,0.000415,0.000179,0,-0.000209,-0.000401,-0.000564,-0.000768,-0.0009,-0.001048,-0.001188,-0.001337,-0.001485
+-0.012545,-0.008151,-0.005587,-0.005116,-0.004502,-0.004023,-0.003299,-0.002743,-0.002242,-0.001745,-0.001207,-0.000737,-0.000317,-0.000101,0.000172,0.000418,0.000583,0.000807,0.000907,0.000785,0.000814,0.000857,0.000611,0.000333,0.000107,0,-0.000184,-0.000386,-0.000511,-0.000777,-0.000878,-0.001006,-0.001164,-0.001298,-0.001468
+-0.013294,-0.008875,-0.006278,-0.005785,-0.005166,-0.004557,-0.003943,-0.003326,-0.002763,-0.002189,-0.001694,-0.001158,-0.000744,-0.000578,-0.000209,2.3e-05,0.000189,0.000503,0.000635,0.000577,0.00062,0.000674,0.000602,0.000254,4e-05,0,-0.000208,-0.00037,-0.000527,-0.000659,-0.000842,-0.00093,-0.001089,-0.001246,-0.00136
```
Comment (asmith-whoi) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20210811__CC_taarray.ext:82]: Good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20210811__CC_tcarray.ext [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+-0.013959,-0.011663,-0.010459,-0.010181,-0.009811,-0.00942,-0.009139,-0.00872,-0.008428,-0.007942,-0.007443,-0.006828,-0.006238,-0.005559,-0.005031,-0.004823,-0.004367,-0.003475,-0.002998,-0.002396,-0.001885,-0.001514,-0.001158,-0.000779,-0.000338,0,0.000237,0.000521,0.000663,0.000779,0.001012,0.001232,0.001314,0.001449,0.001407
+-0.013041,-0.011051,-0.009766,-0.009555,-0.009138,-0.008965,-0.00864,-0.008013,-0.007577,-0.007244,-0.006713,-0.006189,-0.005837,-0.005238,-0.004723,-0.004155,-0.003754,-0.002982,-0.00244,-0.001977,-0.00158,-0.001281,-0.000747,-0.000501,-0.000312,0,0.000181,0.000563,0.000755,0.00083,0.000963,0.001129,0.001155,0.001138,0.001252
+-0.012576,-0.010497,-0.009432,-0.009013,-0.008594,-0.008414,-0.008126,-0.007785,-0.007406,-0.007066,-0.006546,-0.006022,-0.005546,-0.005075,-0.004607,-0.004224,-0.003753,-0.002978,-0.002505,-0.001979,-0.001529,-0.001296,-0.000981,-0.000651,-0.000339,0,9.1e-05,0.000425,0.000554,0.0007,0.000923,0.001098,0.001103,0.001269,0.001172
```
Comment (asmith-whoi) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20210811__CC_tcarray.ext:82]: Good

**Review by asmith-whoi — CHANGES_REQUESTED** [review]
Change underscore to dash in PHSEN source file name on the original file, and then in the csv. See note below.

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20210824.csv [hunk 10]
```diff
@@ -0,0 +1,8 @@
...
...9, 272.82, 273.64, 274.46, 275.28, 276.1, 276.93, 277.75, 278.57, 279.39, 280.21, 281.04, 281.86, 282.68, 283.5, 284.32, 285.15, 285.97, 286.79, 287.61, 288.43, 289.26, 290.08, 290.9, 291.72, 292.54, 293.37, 294.19, 295.01, 295.83, 296.65, 297.47, 298.3, 299.12, 299.94, 300.76, 301.58, 302.4, 303.22, 304.05, 304.87, 305.69, 306.51, 307.33, 308.15, 308.97, 309.79, 310.61, 311.43, 312.26, 313.08, 313.9, 314.72, 315.54, 316.36, 317.18, 318, 318.82, 319.64, 320.46, 321.28, 322.1, 322.92, 323.74, 324.55, 325.37, 326.19, 327.01, 327.83, 328.65, 329.47, 330.29, 331.1, 331.92, 332.74, 333.56, 334.38, 335.19, 336.01, 336.83, 337.65, 338.46, 339.28, 340.1, 340.92, 341.73, 342.55, 343.37, 344.18, 345, 345.81, 346.63, 347.44, 348.26, 349.08, 349.89, 350.71, 351.52, 352.34, 353.15, 353.96, 354.78, 355.59, 356.41, 357.22, 358.03, 358.85, 359.66, 360.47, 361.28, 362.1, 362.91, 363.72, 364.53, 365.34, 366.16, 366.97, 367.78, 368.59, 369.4, 370.21, 371.02, 371.83, 372.64, 373.45, 374.26, 375.07, 375.88, 376.69, 377.49, 378.3, 379.11, 379.92, 380.73, 381.53, 382.34, 383.15, 383.95, 384.76, 385.57, 386.37, 387.18, 387.98, 388.79, 389.59, 390.4, 391.2, 392.01, 392.81, 393.61, 394.42, 395.22, 396.02]",
```
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01091__20210824.csv:8]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01094__20210824.csv [hunk 11]
```diff
@@ -0,0 +1,8 @@
...
..., 271.85, 272.66, 273.47, 274.27, 275.08, 275.89, 276.7, 277.51, 278.32, 279.13, 279.94, 280.75, 281.56, 282.37, 283.18, 283.99, 284.8, 285.61, 286.42, 287.23, 288.03, 288.84, 289.65, 290.46, 291.27, 292.08, 292.89, 293.7, 294.51, 295.32, 296.12, 296.93, 297.74, 298.55, 299.36, 300.17, 300.98, 301.78, 302.59, 303.4, 304.21, 305.02, 305.82, 306.63, 307.44, 308.25, 309.06, 309.86, 310.67, 311.48, 312.29, 313.09, 313.9, 314.71, 315.51, 316.32, 317.13, 317.93, 318.74, 319.55, 320.35, 321.16, 321.97, 322.77, 323.58, 324.38, 325.19, 326, 326.8, 327.61, 328.41, 329.22, 330.02, 330.83, 331.63, 332.44, 333.24, 334.04, 334.85, 335.65, 336.46, 337.26, 338.06, 338.87, 339.67, 340.47, 341.28, 342.08, 342.88, 343.68, 344.49, 345.29, 346.09, 346.89, 347.69, 348.5, 349.3, 350.1, 350.9, 351.7, 352.5, 353.3, 354.1, 354.9, 355.7, 356.5, 357.3, 358.1, 358.9, 359.7, 360.5, 361.29, 362.09, 362.89, 363.69, 364.49, 365.28, 366.08, 366.88, 367.67, 368.47, 369.27, 370.06, 370.86, 371.66, 372.45, 373.25, 374.04, 374.84, 375.63, 376.42, 377.22, 378.01, 378.81, 379.6, 380.39, 381.19, 381.98, 382.77, 383.56, 384.35, 385.15, 385.94, 386.73, 387.52, 388.31, 389.1, 389.89, 390.68, 391.47, 392.26, 393.05, 393.84]",
```
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01094__20210824.csv:8]: Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00661__20210814.csv [hunk 12]
```diff
@@ -0,0 +1,8 @@
...
... 271.26, 272.07, 272.88, 273.69, 274.5, 275.32, 276.13, 276.94, 277.75, 278.56, 279.38, 280.19, 281, 281.81, 282.62, 283.44, 284.25, 285.06, 285.87, 286.68, 287.5, 288.31, 289.12, 289.93, 290.75, 291.56, 292.37, 293.18, 293.99, 294.81, 295.62, 296.43, 297.24, 298.05, 298.87, 299.68, 300.49, 301.3, 302.11, 302.92, 303.74, 304.55, 305.36, 306.17, 306.98, 307.79, 308.61, 309.42, 310.23, 311.04, 311.85, 312.66, 313.47, 314.28, 315.1, 315.91, 316.72, 317.53, 318.34, 319.15, 319.96, 320.77, 321.58, 322.39, 323.2, 324.01, 324.82, 325.63, 326.44, 327.25, 328.06, 328.87, 329.68, 330.49, 331.3, 332.11, 332.92, 333.73, 334.54, 335.34, 336.15, 336.96, 337.77, 338.58, 339.39, 340.19, 341, 341.81, 342.62, 343.43, 344.23, 345.04, 345.85, 346.65, 347.46, 348.27, 349.07, 349.88, 350.69, 351.49, 352.3, 353.1, 353.91, 354.72, 355.52, 356.33, 357.13, 357.94, 358.74, 359.55, 360.35, 361.15, 361.96, 362.76, 363.57, 364.37, 365.17, 365.98, 366.78, 367.58, 368.38, 369.19, 369.99, 370.79, 371.59, 372.39, 373.2, 374, 374.8, 375.6, 376.4, 377.2, 378, 378.8, 379.6, 380.4, 381.2, 382, 382.8, 383.6, 384.39, 385.19, 385.99, 386.79, 387.59, 388.38, 389.18, 389.98, 390.77, 391.57, 392.37, 393.16, 393.96, 394.75]",
```
Comment (reedan88) [inline calibration/NUTNRM/CGINS-NUTNRM-00661__20210814.csv:8]: Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00760__20210226.csv [hunk 13]
```diff
@@ -0,0 +1,8 @@
...
...273.71, 274.52, 275.32, 276.13, 276.94, 277.75, 278.56, 279.37, 280.18, 280.99, 281.8, 282.61, 283.42, 284.23, 285.04, 285.85, 286.65, 287.46, 288.27, 289.08, 289.89, 290.7, 291.51, 292.32, 293.13, 293.94, 294.75, 295.56, 296.37, 297.18, 297.99, 298.8, 299.61, 300.42, 301.23, 302.04, 302.84, 303.65, 304.46, 305.27, 306.08, 306.89, 307.7, 308.51, 309.32, 310.13, 310.94, 311.75, 312.56, 313.37, 314.17, 314.98, 315.79, 316.6, 317.41, 318.22, 319.03, 319.84, 320.64, 321.45, 322.26, 323.07, 323.88, 324.69, 325.49, 326.3, 327.11, 327.92, 328.73, 329.53, 330.34, 331.15, 331.96, 332.76, 333.57, 334.38, 335.19, 335.99, 336.8, 337.61, 338.41, 339.22, 340.03, 340.83, 341.64, 342.45, 343.25, 344.06, 344.86, 345.67, 346.48, 347.28, 348.09, 348.89, 349.7, 350.5, 351.31, 352.11, 352.92, 353.72, 354.53, 355.33, 356.13, 356.94, 357.74, 358.55, 359.35, 360.15, 360.96, 361.76, 362.56, 363.36, 364.17, 364.97, 365.77, 366.57, 367.38, 368.18, 368.98, 369.78, 370.58, 371.38, 372.18, 372.98, 373.79, 374.59, 375.39, 376.19, 376.99, 377.79, 378.58, 379.38, 380.18, 380.98, 381.78, 382.58, 383.38, 384.18, 384.97, 385.77, 386.57, 387.37, 388.16, 388.96, 389.76, 390.55, 391.35, 392.14, 392.94, 393.73, 394.53]",
```
Comment (reedan88) [inline calibration/NUTNRM/CGINS-NUTNRM-00760__20210226.csv:8]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20210729.csv (hunk 4 above)
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20210729.csv:9]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00151__20210804.csv [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+ACS-151,CC_tbins,"[1.331144, 2.415952, 3.477692, 4.426471, 5.537727, 6.505200, 7.481154, 8.498400, 9.485385, 10.489167, 11.499600, 12.508800, 13.503750, 14.491600, 15.501154, 16.499231, 17.509600, 18.487600, 19.490800, 20.486154, 21.470400, 22.480000, 23.488077, 24.508462, 25.457037, 26.484000, 27.576207, 28.461786, 29.523043, 30.488571, 31.496400, 32.480000, 33.486129, 34.536154]",
+ACS-151,CC_taarray,SheetRef:CC_taarray,
+ACS-151,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00151__20210804.csv:9]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20210729__CC_taarray.ext (hunk 5 above)
Comment (reedan88) [reply calibration/OPTAAD/CGINS-OPTAAD-00130__20210729__CC_taarray.ext:83]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20210729__CC_tcarray.ext (hunk 6 above)
Comment (reedan88) [reply calibration/OPTAAD/CGINS-OPTAAD-00130__20210729__CC_tcarray.ext:83]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00151__20210804__CC_taarray.ext [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+-0.011032,-0.008139,-0.007614,-0.007317,-0.007027,-0.006792,-0.00639,-0.005869,-0.005481,-0.005135,-0.004854,-0.004494,-0.004046,-0.003694,-0.003442,-0.00288,-0.002628,-0.002264,-0.001889,-0.001559,-0.001194,-0.000895,-0.000558,-0.000295,0,0.000165,0.000387,0.000587,0.000688,0.000868,0.00102,0.000937,0.001074,0.000964
+-0.011041,-0.008154,-0.007609,-0.007271,-0.007007,-0.006766,-0.006411,-0.005878,-0.005459,-0.005101,-0.004811,-0.004506,-0.004024,-0.003708,-0.003447,-0.002921,-0.002581,-0.002218,-0.001844,-0.001558,-0.001206,-0.00085,-0.000488,-0.000276,0,0.000241,0.000463,0.000638,0.000772,0.000878,0.001031,0.000972,0.001149,0.00106
+-0.011064,-0.008217,-0.007698,-0.007382,-0.007118,-0.006982,-0.006508,-0.005957,-0.005545,-0.005212,-0.00494,-0.004718,-0.004163,-0.003863,-0.003636,-0.003108,-0.002726,-0.002396,-0.002035,-0.00169,-0.001354,-0.000948,-0.000522,-0.000318,0,0.000157,0.000334,0.000551,0.00063,0.000777,0.000938,0.000867,0.000988,0.00096
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/OPTAAD/CGINS-OPTAAD-00151__20210804__CC_taarray.ext:87]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00151__20210804__CC_tcarray.ext [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+-0.010363,-0.005031,-0.004237,-0.003659,-0.002989,-0.002725,-0.002499,-0.002313,-0.002122,-0.001934,-0.001775,-0.001655,-0.001578,-0.001424,-0.001367,-0.001112,-0.000979,-0.000971,-0.000784,-0.000589,-0.000514,-0.000287,-0.000197,-0.00021,0,7.4e-05,2.1e-05,-1.2e-05,8.9e-05,-1.2e-05,-0.000101,-0.000166,-0.000125,-0.000374
+-0.010494,-0.005399,-0.004306,-0.003898,-0.00308,-0.00272,-0.002369,-0.002322,-0.002277,-0.002108,-0.001886,-0.001884,-0.001673,-0.001545,-0.001395,-0.001165,-0.00109,-0.000962,-0.000885,-0.000791,-0.000659,-0.000511,-0.000357,-0.00023,0,3.2e-05,0.000168,6e-06,9.6e-05,0.000101,0.000121,0.000186,-3.5e-05,-0.000345
+-0.010408,-0.005205,-0.004321,-0.003693,-0.002975,-0.002736,-0.002462,-0.002382,-0.002283,-0.002183,-0.001928,-0.001907,-0.001781,-0.001776,-0.001436,-0.001356,-0.001153,-0.001109,-0.000898,-0.000759,-0.000544,-0.000474,-0.000309,-0.000165,0,0.000108,0.000126,0.000126,0.00021,0.000173,0.000129,6.9e-05,9.3e-05,-4.4e-05
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/OPTAAD/CGINS-OPTAAD-00151__20210804__CC_tcarray.ext:87]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20210811.csv (hunk 7 above)
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20210811.csv:9]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20210811__CC_taarray.ext (hunk 8 above)
Comment (reedan88) [reply calibration/OPTAAD/CGINS-OPTAAD-00256__20210811__CC_taarray.ext:82]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0155__20210716.csv [hunk 17]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0155,CC_ea434,17372.0, source_file PHSEN-D_SAMI_pH_SN_P0155_Validation_Files_2021-07-16.zip > Val_Certificate_P0155.pdf
+P0155,CC_ea578,94.1,
+P0155,CC_eb434,2284.0,
+P0155,CC_eb578,38676.0,
+P0155,CC_ind_off,0,
+P0155,CC_ind_slp,1,
+P0155,CC_psal,35,Constant
```
Comment (reedan88) [inline calibration/PHSEND/CGINS-PHSEND-P0155__20210716.csv:8]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20210811__CC_tcarray.ext (hunk 9 above)
Comment (reedan88) [reply calibration/OPTAAD/CGINS-OPTAAD-00256__20210811__CC_tcarray.ext:82]: Looks good.

File: calibration/ADCPAM/CGINS-ADCPAM-44067__20150110.csv [hunk 18]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+644067,CC_scale_factor1,0.61,
+644067,CC_scale_factor2,0.61,
+644067,CC_scale_factor3,0.61,
+644067,CC_scale_factor4,0.61,
```
Comment (reedan88) [inline calibration/ADCPAM/CGINS-ADCPAM-44067__20150110.csv:5]: Looks good.

File: calibration/ADCPAM/CGINS-ADCPAM-49653__20150110.csv [hunk 19]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+649653,CC_scale_factor1,0.61,
+649653,CC_scale_factor2,0.61,
+649653,CC_scale_factor3,0.61,
+649653,CC_scale_factor4,0.61,
```
Comment (reedan88) [inline calibration/ADCPAM/CGINS-ADCPAM-49653__20150110.csv:5]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good to me.

File: calibration/PHSEND/CGINS-PHSEND-P0155__20210716.csv [hunk 20]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0155,CC_ea434,17372.0, source_file PHSEN-D_SAMI_pH_SN_P0155_Validation_Files_2021-07-16.zip > Val_Certificate_P0155.pdf
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply calibration/PHSEND/CGINS-PHSEND-P0155__20210716.csv]: All set!

File: calibration/PHSEND/CGINS-PHSEND-P0155__20210716.csv (hunk 17 above)
Comment (reedan88) [reply calibration/PHSEND/CGINS-PHSEND-P0155__20210716.csv:8]: New source file name looks good!

**Review by reedan88 — APPROVED** [review]
Edit to PHSEN source file name looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0155__20210716.csv [hunk 21]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0155,CC_ea434,17372.0, source_file PHSEN-D_SAMI-pH_SN_P0155_Validation_Files_2021-07-16.zip > Val_Certificate_P0155.pdf
+P0155,CC_ea578,94.1,
+P0155,CC_eb434,2284.0,
+P0155,CC_eb578,38676.0,
+P0155,CC_ind_off,0,
+P0155,CC_ind_slp,1,
+P0155,CC_psal,35,Constant
```
Comment (asmith-whoi) [inline calibration/PHSEND/CGINS-PHSEND-P0155__20210716.csv:8]: Source file name good

**Review by asmith-whoi — APPROVED** [review]
Correction noted and approved

**Pull request merged**